ambry-api/src/main/java/com/github/ambry/server/AccountStatsStore.java

```diff
@@ -62,37 +61,123 @@
    *     }
    *   }
    * </pre>
-   * @param clusterName The clusterName.
    * @return The map that represents the container storage usage.
    * @throws Exception
    */
-  Map<String, Map<String, Long>> queryAggregatedStats(String clusterName) throws Exception;
+  Map<String, Map<String, Long>> queryAggregatedAccountStats() throws Exception;
 
   /**
-   * Return the monthly aggregated stats. This method returns a map in the same format as the {@link #queryAggregatedStats}.
+   * Return the monthly aggregated stats. This method returns a map in the same format as the {@link #queryAggregatedAccountStats}.
    * The only difference these two methods have is that this method's returned value only changes in the beginning of each
    * month. For every new month(in local zone offset), an aggregated stats will be written to storage and a snapshot will
    * be created. This method will return current snapshot. This method doesn't require a month value to fetch the snapshot
    * as this new snapshot will be override the old ones.
-   * @param clusterName The clusterName.
-   * @return The map thtat represents the container storage usage.
+   * @return The map that represents the container storage usage.
    * @throws Exception
    */
-  Map<String, Map<String, Long>> queryMonthlyAggregatedStats(String clusterName) throws Exception;
+  Map<String, Map<String, Long>> queryMonthlyAggregatedAccountStats() throws Exception;
 
   /**
    * Return the month value of the current container storage snapshot.
-   * @param clusterName The clusterName.
    * @return The month value for current snapshot, like "2020-01". Empty string will be returned if there is no record.
    * @throws Exception
    */
-  String queryRecordedMonth(String clusterName) throws Exception;
+  String queryRecordedMonth() throws Exception;
 
   /**
    * Taking a snapshot of current aggregated stats and update the month value.
-   * @param clusterName The clusterName.
    * @param monthValue The month in string format, like "2020-01".
    * @throws Exception
    */
-  void takeSnapshotOfAggregatedStatsAndUpdateMonth(String clusterName, String monthValue) throws Exception;
+  void takeSnapshotOfAggregatedAccountStatsAndUpdateMonth(String monthValue) throws Exception;
+
+  /**
+   * Return all the partition class names and their corresponding partition ids in a set.
+   * @return A map whose key is the partition class name and the value is a set of a partition ids.
+   * @throws Exception
+   */
+  Map<String, Set<Short>> queryPartitionNameAndIds() throws Exception;
```

Comment:
It might be good to flesh out the docs for what exceptions can occur in these methods (ideally make the declarations more specific, if possible).  It's fairly opaque.  This can be a follow up.